packages/lesswrong/server/migrations/meta/utils.ts

```diff
@@ -0,0 +1,11 @@
+import PgCollection from "../../../lib/sql/PgCollection";
+import AddFieldQuery from "../../../lib/sql/AddFieldQuery";
+
+export const addField = async <T extends DbObject>(
+  db: SqlClient,
+  collection: PgCollection<T>,
+  fieldName: keyof T & string,
+): Promise<void> => {
+  const {sql} = new AddFieldQuery(collection.getTable(), fieldName).compile();
+  await db.none(sql);
+}
```

Comment:
I see you decided to further reduce boilerplate

Reply:
Yeah - I figured something like this would eventually happen, but I didn't want to randomly add it before we actually had any migrations to use it.